src/unisparks/button.cpp

```diff
@@ -72,30 +72,15 @@ uint8_t buttonStatuses[NUMBUTTONS];
 
 static constexpr uint32_t lockDelay = 10000;
 
-uint8_t brightnessCursor = 0;
+const uint8_t brightnessList[] = { 2, 4, 8, 16, 32, 64, 128, 255 };
+#define NUM_BRIGHTNESSES (sizeof(brightnessList) / sizeof(brightnessList[0]))
+#define MAX_BRIGHTNESS (NUM_BRIGHTNESSES-1)
 
-#ifndef BRIGHTER
-#  define BRIGHTER 1
-#endif // BRIGHTER
-
-// this ordering is silly but we need the highest to be last
-// in case it blows the fuse in the USB battery pack
-#if START_DIM
-#  ifndef FIRST_BRIGHTNESS
-#  define FIRST_BRIGHTNESS 1
-#  endif // FIRST_BRIGHTNESS
-const uint8_t brightnessList[] = { FIRST_BRIGHTNESS, 64, 96, 8, 16, 128};
-#elif BRIGHTER
-#  ifndef FIRST_BRIGHTNESS
-#  define FIRST_BRIGHTNESS 64
-#  endif // FIRST_BRIGHTNESS
-const uint8_t brightnessList[] = {FIRST_BRIGHTNESS, 96, 8, 16, 32, 128};
-#else // BRIGHTER
 #  ifndef FIRST_BRIGHTNESS
-#  define FIRST_BRIGHTNESS 32
+#  define FIRST_BRIGHTNESS 3
```

Comment:
Since we want the original brightness to be 64, should FIRST_BRIGHTNESS be 5?

also, the indentation for `# ifndef FIRST_BRIGHTNESS` and the corresponding `#  endif` isn't right now that we've removed the surrounding `#if`